gravityflow/gravityflowformconnector
# Add support for the step attribute

This PR adds support for the step merge tag attribute. This is useful for feed settings such as Twilio and Slack when the merge tag is replaced while the workflow is currently on the feed step.
Addresses [HS6839](https://secure.helpscout.net/conversation/669665952/6839?folderId=516732)

**Testing Instructions**
1. Create a form with a Twilio feed followed by a form submission step.
2. Put the form submission merge tag in the Twilio message body with the assignee and step attributes set for the Form Submission step. {workflow_form_submission_url: assignee=email_field|3 step=3}
3. Check with different types of assignees e.g. 
{workflow_form_submission_url: assignee=assignee_field|3 step=3}
{workflow_form_submission_url: assignee=assignee_user_field|3 step=3}
{workflow_form_submission_url: assignee=user_id|3 step=3}
3. Check for regressions.
